test/contains_properties_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+require "openxml/element"
+require "openxml/properties"
+require "openxml/contains_properties"
+
+class ContainsPropertiesTest < Minitest::Test
+  should "allow properties to be rendered as direct children" do
+    element = Class.new(OpenXml::Element) do
+      include OpenXml::ContainsProperties
+      tag :bodyPr
+      namespace :a
+
+      value_property :string_property
+    end.new
+    element.string_property = "A Value"
+
+    require "pry"
```

Comment:
Did you mean to leave `require "pry"` in there?

Reply:
Oops! Nope, I'll take that out.